platform/commonUI/edit/src/actions/EditAndComposeAction.js

```diff
@@ -54,9 +54,13 @@ define(
                         .then(doPersist);
             }
 
+            if (this.domainObject.getCapability('type').getKey() !== 'folder') {
+                this.domainObject.getCapability('action').perform('edit');
+            }
```

Comment:
I think I just misread the diff, sorry about that.

The check for edit would make sense (if some future object type also didn't have an edit mode, but did accept drops, I believe folder-like behavior would be appropriate.) But I also see that Folders _do_ support the Edit action (just not in the `view-control` area) due to the way EditActionPolicy is implemented.

That policy could be modified fairly simply to get the intended results, but my inclination is to leave well enough alone and proceed with merge. Thoughts?